src/app/app.configuration.js

```diff
@@ -0,0 +1,20 @@
+(function () {
+    'use strict';
+
+    angular.module( 'roboconf' )
```

Comment:
I will probably try to stick to your coding conventions.
I will have a look at your configuration files.